app/models/cookbook.rb

```diff
@@ -1,4 +1,5 @@
 class Cookbook < ActiveRecord::Base
+  include Badgeable
```

Comment:
I don't think so. Including the concern is what gives an instance of cookbook the methods called on it in the view for showing badges (e.g. `.badges`, `.is? :partner`).